server/src/index.ts

```diff
@@ -224,7 +225,9 @@ const typeDefs = gql`
   input CreateAppGithubInput {
     name: String!
     gitRepoUrl: String!
```

Comment:
as we are passing the id we can probably remove this no?

Reply:
@pradel I think passing full name instead of `gitRepoUrl` makes more sense as that's what we use later on the server side. 

Full name is in format  `userName/repoName`